src/lib/viewers/BaseViewer.js

```diff
@@ -1100,6 +1101,18 @@ class BaseViewer extends EventEmitter {
         this.emit('annotatorevent', data);
     }
 
+    handleAnnotationCreateEvent({ annotation, meta: { status } = {} }) {
+        if (status !== 'pending') {
+            const activeMode = this.annotationControls.getActiveMode();
+            this.annotator.toggleAnnotationMode(activeMode);
```

Comment:
Should we do this inside box-annotations?

Reply:
Given how we're trying to balance box-annotations being a full blown app vs sdk, I think I would lean towards this logic to be determined by the users of the SDK